spinn_front_end_common/interface/interface_functions/local_tdma_builder.py

```diff
@@ -150,6 +164,65 @@ def __call__(
                     machine_time_step, time_scale_factor, fraction_of_waiting)
                 app_vertex.set_initial_offset(initial_offset)
 
+    @staticmethod
+    def _auto_config_times(
+            app_machine_quantity, time_between_cores, fraction_of_sending,
+            time_scale_factor, machine_time_step, application_graph,
+            n_keys_map, machine_graph, fraction_of_waiting):
+
+        # initial offset in time for first core for beginning of tdma
+        initial_offset_point = int(math.ceil(
+            (machine_time_step * time_scale_factor) * fraction_of_waiting))
+
+        # find worst combo (NOTE: it might not exist as an actual app vertex,
+        # but least it'll cover all the things in between)
+        max_cores = 0
+        max_phases = 0
+        for app_vertex in application_graph.vertices:
+            if isinstance(app_vertex, TDMAAwareApplicationVertex):
+                cores = app_vertex.get_n_cores()
+                phases = app_vertex.find_n_phases_for(
+                    machine_graph, n_keys_map)
+                max_cores = max(max_cores, cores)
+                max_phases = max(max_phases, phases)
+        logger.info(
+            "max cores {} and max phases {}".format(max_cores, max_phases))
```

Comment:
This won't mean much to most users; it could either be moved to debug, or if you think it could be made useful to users, update to explain what is happening here.  It should also start with a capital letter.

Reply:
was menat to be debug. had it as info for just chekcing it worked. switched back to debug